lms/views/admin/organization.py

```diff
@@ -44,6 +44,7 @@ def __init__(self, request):
         route_name="admin.organizations",
         request_method="GET",
         renderer="lms:templates/admin/organization/search.html.jinja2",
+        permission=Permissions.STAFF,
```

Comment:
Does this work? I didn't think you could put one permission in the `@view_defaults` and then a different one in the `@view_config`

Reply:
Yeah, seems to work fine :+1: